pkg/services/accesscontrol/models.go

```diff
@@ -310,6 +310,7 @@ func (cmd *SaveExternalServiceRoleCommand) Validate() error {
 			continue
 		}
 		dedupMap[cmd.Permissions[i]] = true
+		cmd.Permissions[i].Kind, cmd.Permissions[i].Attribute, cmd.Permissions[i].Identifier = SplitScope(cmd.Permissions[i].Scope)
```

Comment:
Feels a bit weird for the scope split to be performed as part of the validation step. I'm fine to leave it here if it is the most convenient place, but I wouldn't normally expect the input of validation command to change as part of the validation process. That said, we already change it by deduplicating permissions.